### PR TITLE
Fix color

### PR DESCRIPTION
**Pull request by DantongWang**
Color for first cluster should be fixed now, something like dark blue. Used grey for 1-size clusters.

### CODE REVIEW

**Comment by yannikschaelte** [comment]
in the notebook, there is still is inf error in the visualization part. does the notebook work on your computer?

**Comment by yannikschaelte** [comment]
no, just checked, if I run it, everything is fine. could you re-run the notebooks and upload them?

**Comment by dweindl** [comment]
I will follow @yannikschaelte 's review

File: pypesto/visualize/clust_color.py [hunk 1]
```diff
@@ -74,14 +74,23 @@ def assign_clustered_colors(vals):
...
-    col = scalarmap.to_rgba(ind_col)
+    # grey color for 1-size cluster
+    pre_col[vmax] = (0.7, 0.7, 0.7, 1)
+    ind_col = np.zeros(len(clust))
+    sum_col = 0
+    for iind, value_ind in enumerate(ind_clust):
+        if clustsize[value_ind] > 1:
+            ind_col[iind] = int(sum_col)
+            if iind < len(ind_clust) - 1:
+                if value_ind != ind_clust[iind + 1]:
```
Comment (yannikschaelte) [inline pypesto/visualize/clust_color.py]: I don't understand what this is for. Can you add some description (``# comments``)?

Comment (DantongWang) [reply pypesto/visualize/clust_color.py]: Okay, a little bit hard to explain... will try.

**Comment by DantongWang** [comment]
hope the comments are helpful

File: pypesto/visualize/clust_color.py (hunk 1 above)
Comment (yannikschaelte) [reply pypesto/visualize/clust_color.py]: much better 👍 looks god.

**Pull request merged**